Cubical/HITs/ListedFiniteSet/Properties.agda

```diff
@@ -2,38 +2,103 @@
 module Cubical.HITs.ListedFiniteSet.Properties where
 
 open import Cubical.Foundations.Prelude
+open import Cubical.Data.Prod using (_×_; _,_)
 
 open import Cubical.HITs.ListedFiniteSet.Base
 
 private
   variable
-    A : Type₀
-
-_++_ : ∀ (xs ys : LFSet A) → LFSet A
-[]                  ++ ys = ys
-(x ∷ xs)            ++ ys = x ∷ (xs ++ ys)
----------------------------------------------
-dup x xs i          ++ ys = proof i
-   where
-     proof :
-     -- Need:  (x ∷ x ∷ xs) ++ ys ≡ (x ∷ xs) ++ ys
-     -- which reduces to:
-               x ∷ x ∷ (xs ++ ys) ≡ x ∷ (xs ++ ys)
-     proof = dup x (xs ++ ys)
-comm x y xs i       ++ ys = comm x y (xs ++ ys) i
-trunc xs zs p q i j ++ ys
-  = trunc (xs ++ ys) (zs ++ ys) (cong (_++ ys) p) (cong (_++ ys) q) i j
-
+    ℓ : Level
+    A B : Type ℓ
 
 assoc-++ : ∀ (xs : LFSet A) ys zs → xs ++ (ys ++ zs) ≡ (xs ++ ys) ++ zs
 assoc-++ []       ys zs = refl
-assoc-++ (x ∷ xs) ys zs
-  = cong (x ∷_) (assoc-++ xs ys zs)
-------------------------------------
-assoc-++ (dup x xs i) ys zs j
-  = dup x (assoc-++ xs ys zs j) i
-assoc-++ (comm x y xs i) ys zs j
-  = comm x y (assoc-++ xs ys zs j) i
-assoc-++ (trunc xs xs' p q i k) ys zs j
-  = trunc (assoc-++ xs ys zs j) (assoc-++ xs' ys zs j)
-          (cong (\ xs -> assoc-++ xs ys zs j) p) (cong (\ xs -> assoc-++ xs ys zs j) q) i k
+assoc-++ (x ∷ xs) ys zs = cong (x ∷_) (assoc-++ xs ys zs)
+assoc-++ (dup x xs i) ys zs j = dup x (assoc-++ xs ys zs j) i
+assoc-++ (comm x y xs i) ys zs j = comm x y (assoc-++ xs ys zs j) i
+assoc-++ (trunc xs xs' p q i k) ys zs j = trunc
```

Comment:
I think `trunc` should go on the next line with the arguments indented more. It's currently kind of weird and hard to read because the function is further to the right than its arguments.

The same applies to the calls to `PropElim.f` in the functions below